.github/workflows/node-ci.yml

```diff
@@ -148,6 +148,7 @@ jobs:
 
             - name: Upload Cypress Screenshots
               uses: actions/upload-artifact@v2
+              if: failure()
```

Comment:
This was added because this step wasn't getting run in cases where our `yarn cypress:ci` step failed... which is exactly when we want these screenshots uploaded. I think it's worthwhile to leave this in place. I'm pretty sure we won't get screenshots on `success()` and only on `failure()`.

I started debugging the Cypress failures (which was part of why I started pulling on this `KaTeXErrorView` component in the first place).